### ISSUE TITLE
taint: add @psalm-taint-escape html for `Js::from()` and `Js::encode()`

### ISSUE THREAD

**Issue by alies-dev** [psalm/laravel-psalm-plugin#533]
## Problem

`Illuminate\Support\Js::from()` and `Js::encode()` encode data with mandatory HTML-safe JSON flags (`JSON_HEX_TAG | JSON_HEX_APOS | JSON_HEX_AMP | JSON_HEX_QUOT`). The Blade `@js()` directive compiles to `Js::from($expression)->toHtml()`.

Without taint escape annotations, `@js($userInput)` triggers false-positive taint errors even though the output is safe for embedding in HTML/JS contexts.

## Current state

No stub exists for `Illuminate\Support\Js` in either `stubs/common/` or `stubs/taintAnalysis/`.

## Proposed fix

Create `stubs/taintAnalysis/Support/Js.stubphp`:

```php
namespace Illuminate\Support;

class Js
{
    /**
     * @psalm-taint-escape html
     * @psalm-taint-escape has_quotes
     */
    public static function from($data, $flags = 0, $depth = 512): static {}

    /**
     * @psalm-taint-escape html
     * @psalm-taint-escape has_quotes
     */
    public static function encode($data, $flags = 0, $depth = 512): string {}
}
```

## Methods

- `Js::from($data, $flags, $depth): static` — `Illuminate\Support\Js`
- `Js::encode($data, $flags, $depth): string` — `Illuminate\Support\Js`

## Impact

High — prevents false positives when using `@js()` Blade directive with user data.
